### ISSUE TITLE
Check for update bad design

### ISSUE THREAD

**Issue by 532910** [vector-im/riot-web#11313]
`Settings` -> `Help & About` -> `Check for update`
This button produces the header that is imperceptible, on the background layer, dimmed, and disabled (you can take action only after you close settings). If you don't know where it should appear it looks like the `Check for update` doesn't work at all.

**Comment by t3chguy** [comment]
This probably needs design, simplest thing we could do is have the Settings close when you press it so you see the bar more clearly

**Comment by 532910** [comment]
The button should be `Check for update` -> `new update found: Install` / `nothing to update: Recheck`

**Comment by aaronraimist** [comment]
Duplicate of #10393
